accelerometer/summariseEpoch.py

```diff
@@ -121,6 +124,14 @@ def getActivitySummary(epochFile, nonWearFile, summary,
     if intensityDistribution:
         calculateECDF(e, 'acc', summary)
 
+    # Calculate circadian metrics
+    if psd:
+        calculatePSD(e, epochPeriod, fourierWithAcc, labels, summary)
```

Comment:
create a new circadianRhythms.py file within accelerometer/ directory for the methods containing the code for these sections i.e. make it like the accClassification.py file and how we call e.g. accClassification.activityClassification(e, activityModel)

This would make the summariseEpoch.py file less long, and also contains these methods in a fairly logical order

Reply:
resolved in 5dba99c42e5ce4eac5766d7560433e6546c839f7